src/main/java/space/obminyashka/items_exchange/service/impl/UserServiceImpl.java

```diff
@@ -134,7 +134,7 @@ public String updateUserPassword(UserChangePasswordDto userChangePasswordDto, Us
 
     @Override
     public String updateUserEmail(UserChangeEmailDto userChangeEmailDto, User user) {
```

Comment:
It would be much better to refactor that method and logic instead of continuing to do the same stuff.
1. The user's email could be updated on the controller's layer
2. You can create a very single method update(User user) that would save the new data and nothing more
3. After that response message could be returned from the controller.

Reply:
Done